webapp/src/app/api/funding.ts

```diff
@@ -1,9 +1,29 @@
 "use client";
 
 import { ChainAddress } from '@/app/types/chainAddress';
-import { FundingResData } from '@/app/types/fundingRecord';
+import { FundingResData, FundingFlowRecord } from '@/app/types/fundingRecord';
 
 
-export const getFundingGraph = async (source: ChainAddress): Promise<FundingResData> => {
+export const getFundingGraph = async (): Promise<FundingResData> => {
+    const mapEdgeToFlowRecord = (edge): FundingFlowRecord => {
```

Comment:
It is better not to use `as` if it is not needed.
Even if the model changes, the compiler will not throw an error

Reply:
agree, better way probably is to implement the interface